test/conv_common.hpp

```diff
@@ -2143,7 +2145,9 @@ struct conv_driver : test_driver
                                                       miopen::conv::Direction::Forward);
                 ctx.SetStream(&get_handle());
 
-                bool skip_forward = (input.desc.GetType() == miopenInt8x4 && !IsGemmAplicable(ctx));
+                // TODO: Check if this is still valid
+                bool skip_forward = false; // is_int8 && !IsGemmAplicable(ctx);
```

Comment:
[Recommendation] Good catch! Yes, Naive Solvers should do the job. I think that the comments may be removed.

Reply:
Resolved!